packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -124,62 +122,119 @@ export const createHandler = async ({ input, ctx }: CreateOptions) => {
         webappIPAddress: await getIPAddress(
           WEBAPP_URL.replace("https://", "")?.replace("http://", "").replace(/(:.*)/, "")
         ),
-        t,
+        t: loggedInUserTranslation,
       });
     } else {
       console.warn("Organization created: subdomain not configured and couldn't notify adminnistrators");
     }
   }
 
   const autoAcceptEmail = orgOwnerEmail.split("@")[1];
-  const nonOrgUsernameForOwner = orgOwner.username || "";
-  const { organization, ownerProfile } = await OrganizationRepository.createWithOwner({
-    orgData: {
-      name,
-      slug,
-      isOrganizationConfigured,
-      isOrganizationAdminReviewed: IS_USER_ADMIN,
-      autoAcceptEmail,
-      seats: seats ?? null,
-      pricePerSeat: pricePerSeat ?? null,
-      isPlatform,
-    },
-    owner: {
-      id: orgOwner.id,
+
+  const orgData = {
+    name,
+    slug,
+    isOrganizationConfigured,
+    isOrganizationAdminReviewed: IS_USER_ADMIN,
+    autoAcceptEmail,
+    seats: seats ?? null,
+    pricePerSeat: pricePerSeat ?? null,
+    isPlatform,
+  };
+
+  // Create a new user and invite them as the owner of the organization
+  if (!orgOwner) {
+    const data = await OrganizationRepository.createWithNonExistentOwner({
+      orgData,
+      owner: {
+        email: orgOwnerEmail,
+      },
+    });
+
+    orgOwner = data.orgOwner;
+
+    const { organization, ownerProfile } = data;
+
+    const translation = await getTranslation(input.language ?? "en", "common");
+
+    sendEmailVerification({
```

Comment:
```suggestion
    await sendEmailVerification({
```

Reply:
Oh great catch @joeauyeung!! It was working locally but it would not have worked in production I believe as it is serverless.